docker-compose.yml

```diff
@@ -27,12 +27,15 @@ services:
   backend:
     restart: always
     container_name: pesticide_backend
-    command: bash -c "cd ./src && python check_db.py --service-name db --ip db --port 3306 && python manage.py collectstatic --noinput && python manage.py makemigrations pesticide_app && python manage.py makemigrations && python manage.py migrate && daphne -b 0.0.0.0 -p 8000 pesticide.asgi:application"
+    command: bash -c "cd ./src && python check_db.py --service-name db --ip db --port 3306 && python manage.py collectstatic --noinput && daphne -b 0.0.0.0 -p 8000 pesticide.asgi:application"
```

Comment:
Do not commit this modified docker-compose file. The modifications made were only for easy development, and cannot be used in a production setup.